sql/2025/fonts/execute.py

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/env python
```

Comment:
Is this `font` specific? Or should we put it in the `sql/util` folder?

Reply:
I tried to keep it general, and it could be useful for others. However, I am relying on the standardized header format (see the readme) for exporting the context to the spreadsheet. To make it 100% general, one would have to treat the header's absence as a special case.